src/sprout/Helpers/Constants.php

```diff
@@ -96,11 +96,17 @@ public static function reverseLookup($class, $prefix, $value)
     **/
     public static $recent_dates = array(
         '1 DAY' => 'Last 24 hours',
+        '1 WEEK' => 'Last 7 days',
+        '1 MONTH' => 'Last 30 days',
+        '3 MONTH' => 'Last 3 months',
+        '1 YEAR' => 'Last 12 months',
         'YESTERDAY' => 'Yesterday',
-        '1 WEEK' => 'This week',
-        '1 MONTH' => 'This month',
-        '3 MONTH' => 'This quarter',
-        '1 YEAR' => 'This year',
+        'THIS_WEEK' => 'This week',
+        'THIS_MONTH' => 'This month',
+        'THIS_YEAR' => 'This year',
+        'WEEK_TO_DATE' => 'Week to date',
+        'MONTH_TO_DATE' => 'Month to date',
+        'YEAR_TO_DATE' => 'Year to date',
```

Comment:
I do fear overcrowding this but I wonder there's room for a set of LAST_WEEK/MONTH/YEAR ?

Or perhaps there's a way we can make this pluggable so we can customise these ranges per site. That's probably a better idea.

Reply:
I thought about that, but I couldn't think of anything off the top of my head and was a bit time poor to be hunting for it..

Happy to add it in if @aitken85 or anyone else can think of something